### DISCUSSION TITLE
"Ho'oleilana: An Individual Baryon Acoustic Oscillation?"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#136] [Redshift Models]
"Ho'oleilana: An Individual Baryon Acoustic Oscillation?"
Open Access [https://iopscience.iop.org/article/10.3847/1538-4357/aceaf3](https://iopscience.iop.org/article/10.3847/1538-4357/aceaf3)

*<<The interpretation of Ho'oleilana as a BAO structure with our preferred analysis
implies a value of the Hubble constant of* $76.9_{-4.8}^{+8.2}$ *km/s/Mpc>>*

**Comment by HanDeBruijn** [comment]
Don't know what a _Baryon Acoustic Oscillation_ is, but anyway the value [by Riess et al.](https://github.com/orgs/a-cosmology-group/discussions/117#discussioncomment-6631882) $73.4$ _km/s/Mpc_ is in that range.
